Common.prf

```diff
@@ -18,6 +18,9 @@ CONFIG(release, debug|release) {
   QMAKE_CXXFLAGS += -O3
 }
 
+# Enable C++14
+CONFIG += c++14
+
```

Comment:
I guess this is required for the reserve functions. So this means that we can no longer build HopsanCore with old compilers? (not a problem, since we already decided on this, I just want to make sure).

Reply:
No this is completely unrelated, I dont remember the problem exactly, but I got linking problems due to not all parts of Hopsan using the same C++ standard version (maybe on Linux). 

Anyway, since we do "Rebase and Merge" (I do ff-only merge to keep my commit signatures) I have no problems adding some extra unrelated commits to PR like this. since they do not actually add new functionality or such things, its only cleanup och the build system files.
But I guess it would have been more "correct" to put them in a separate PR.

Reply:
Ok, then I understand. In general I prefer a separate PR for everything, but I guess some minor changes does not really matter. It is more important in projects that enforce squashing all commits when merging a PR.